### DISCUSSION TITLE
How to install libpqxx on arm mac?

### DISCUSSION THREAD

**Discussion by D0zee** [microsoft/vcpkg#40553] [Q&A]
Hello everyone! I believe this question is resolved somewhere, I just can't find solution. I try to migrate from `conan` to `vcpkg` and have encountered problem with `libpqxx` installing.

The error is:

```
error: building openssl:arm64-osx failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle openssl:arm64-osx: 14 s
```

I would be very glad if somebody has already met this issue and knows how to sort it out. Thank you in advance!

**Comment by dg0yt** [comment]
Is that error still for openssl 1.1.1i as before the edit?

**Reply by D0zee** [reply]
Hey, no, it turned out that 1.1.1i is not error. Even libpqxx can be installed on mac. The mistake was wrong triplet configuration that must be explicitly set in vs code setting file. I will close this discussion. Thank you!
